tools/azure-pipelines/build-apache-repo.yml

```diff
@@ -55,6 +55,7 @@ variables:
   SECRET_S3_BUCKET: $[variables.IT_CASE_S3_BUCKET]
   SECRET_S3_ACCESS_KEY: $[variables.IT_CASE_S3_ACCESS_KEY]
   SECRET_S3_SECRET_KEY: $[variables.IT_CASE_S3_SECRET_KEY]
+  SECRET_GE_ACCESS_KEY : $[variables.GE_ACCESS_KEY]
```

Comment:
To whoever reviews this, this is an access key that can be provided by the ASF Infra team directly to an individual that can set it in your Azure DevOps organization.